api/nodes/serializers.py

```
@@ -25,6 +25,33 @@ def to_internal_value(self, data):
         return data
 
 
+class JSONAPINodeListSerializer(JSONAPIListSerializer):
+    """
+    Bulk updates instance with the validated data.
+
+    Request either completely succeeds or fails. Requires
+    the request to be in the serializer context.
+    """
+
+    # Overrides JSONAPIListSerialize which doesn't support multiple update by default.
+    def update(self, instance, validated_data):
+        data_mapping = {item.get('_id', None): item for item in validated_data}
+        request = self.context['request']
+        auth = Auth(request.user)
+        ret = []
+        for node_id, data in data_mapping.items():
```

Should we raise an exception if `node_id == None`?